src/main/java/games/strategy/engine/framework/GameRunner.java

```diff
@@ -139,9 +125,12 @@ public static void main(final String[] args) {
       usage();
       return;
     }
-    SwingUtilities.invokeLater(LookAndFeel::setupLookAndFeel);
 
-    HttpProxy.setupProxies();
+    LookAndFeel.setupLookAndFeel();
```

Comment:
This one was a bit interesting, `setupLookAndFeel` is an `invokeAndWait` call, so no point in wrapping it in an `invokeLater` call.